opta/core/kubernetes.py

```diff
@@ -273,6 +273,22 @@ def update_secrets(namespace: str, secret_name: str, new_values: dict) -> None:
     v1.replace_namespaced_secret(secret_name, namespace, current_secret_object)
 
 
+def remove_secrets(namespace: str, secret_name: str, entry_name: str) -> None:
+    """
+    remove secret from secret store.
+
+    create the secret if it doesn't exist yet.
+    """
+    load_opta_kube_config()
+    v1 = CoreV1Api()
+    create_secret_if_not_exists(namespace, secret_name)
```

Comment:
Why are we creating a secret resource if we are removing a secret? Shouldn't we return early if it doesn't exist?